html-forms/index.html

```diff
@@ -0,0 +1,34 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <title>HTML Forms</title>
+  </head>
+  <body>
+    <form action="index.html" method="GET">
+      <h1>Coding Form!</h1>
+      <div><label for="e-mail">Email</label></div>
+      <input type="email" id="e-mail">
+
+      <div>Why is coding cool?</div>
+      <select name="dropdown">
+        <option>Select an option</option>
+        <option>Because it's interesting</option>
+        <option>Because it's fun</option>
+        <option>Because it's powerful</option>
+      </select>
+
+      <div>Favorite Language(Choose one)</div>
+      <input type="radio" name="radio-button" value="HTML"/> HTML
```

Comment:
For radio inputs, the `label` should surround the text describing the radio input's value, which would here be the text "HTML". Make sure to give each label a `for` attribute and an individual `id` for each radio input 👍